config/crd/bases/osp-director.openstack.org_openstackbackuprequests.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.8.0
+    controller-gen.kubebuilder.io/version: v0.9.0
```

Comment:
Didn't we run into some sort of issue with `v0.9.0` with Podified such that we immediately bumped to `v0.9.2`?  If so, just wondering if we want to do that here too

Reply:
I'll bump it. Wasn't strictly required for this update but yes we've done it elsewhere and won't harm anything.

Reply:
I had a cached version in bin/controller-gen. Removed and re-ran make. All good now I think